checkov/secrets/runner.py

```diff
@@ -128,6 +128,22 @@ def run(
                     'limit': ENTROPY_KEYWORD_LIMIT
                 }
             ]
+        custom_plugins = os.getenv("CHECKOV_CUSTOM_DETECTOR_PLUGINS_PATH", None)
+        logging.info(f"Custom detector flag set to {custom_plugins}")
+        if custom_plugins:
+            detector_path = f"{custom_plugins}/platform_regex_detector.py"
+            file_exists = exists(detector_path)
+            if file_exists:
+                logging.info(f"Custom detector found at {detector_path}. Loading...")
+                plugins_used.append({
+                    'name': 'PlatformRegexDetector',
```

Comment:
Does that mean that all the plugins we add in this way would be named `PlatformRegexDetector`?

Reply:
No, that's just the name of the common infra plugin I created. Each detector plugin will have his own name and will be loaded by platform_regex_detector itself.

Reply:
🔥